src/sidebar/services/test/annotations-exporter-test.js

```diff
@@ -0,0 +1,29 @@
+import { AnnotationsExporter } from '../annotations-exporter';
+
+describe('AnnotationsExporter', () => {
+  let fakeStore;
+  let exporter;
+
+  beforeEach(() => {
+    fakeStore = {
+      profile: sinon.stub().returns({ userid: 'userId' }),
+      allAnnotations: sinon.stub(),
+    };
+    exporter = new AnnotationsExporter(fakeStore);
+  });
+
+  it('generates export content with provided annotations', () => {
+    const now = new Date();
+    const annotations = [{}, {}];
```

Comment:
Given that we're about to build on this work to define what fields do get delivered, this is OK for now; we should use fixtures at some point as this feature gets more defined.

Reply:
This test is a bit improved in https://github.com/hypothesis/client/pull/5644, where the property cleanup implementation is introduced.

I'll make sure I do something even closer to reality there.